# DP: Backport PR lto/62032.

gcc/lto/

2014-08-15  Bin Cheng  <bin.cheng@arm.com>

	Backport from mainline
	2014-08-08  Bin Cheng  <bin.cheng@arm.com>

	PR lto/62032
	* lto-lang.c (lto_init): Switch mis-matched arguments.

--- a/gcc/lto/lto-lang.c
+++ b/gcc/lto/lto-lang.c
@@ -1185,8 +1185,8 @@
     }
   else
     {
-      lto_define_builtins (va_list_type_node,
-			   build_reference_type (va_list_type_node));
+      lto_define_builtins (build_reference_type (va_list_type_node),
+			   va_list_type_node);
     }
 
   targetm.init_builtins ();
